_data/current.yml

```diff
@@ -197,6 +197,13 @@
   twitter: testing_united
   status: <a href="https://www.testingunited.com/become-a-speaker-2025/?utm_source=testingconferences" target="_blank">CFP is Open</a> until February 28, 2025
 
+- name: Tokyo Test Fest
+  location: Tokyo, Japan
+  dates: "November 14, 2025"
+  url: https://www.tokyotestfest.com/?utm_source=testingconferences
+  twitter: tokyotestfest
+  status: <a href="https://forms.gle/3poLaowqAXM4xhNm8" target="_blank">CFP is Open</a> | <a href="https://fienta.com/tokyotestfest2025/?utm_source=testingconferences" target="_blank">Registration is Open</a>
```

Comment:
Actually I think it failed due to the pipe between CFP Open and Registration Open. I put up a fix.